arduino/IoTkit/examples/IoTKitQuickStartExample/AdvancedSetup.ino

```diff
@@ -0,0 +1,109 @@
+#include <Wire.h>
+#include "rgb_lcd.h" //library from https://github.com/Seeed-Studio/Sketchbook_Starter_Kit_V2.0/tree/master/libraries/Grove_LCD_RGB_Backlight
```

Comment:
Should we include this library so that sketch can be run just after downloading?